pkg/rid/store/cockroach/store_test.go

```diff
@@ -26,7 +26,7 @@ var (
 )
 
 func init() {
-	DefaultTimeout = 50 * time.Millisecond
+	DefaultTimeout = 500 * time.Millisecond
```

Comment:
Is this change related to a race condition in `test-go-units-crdb` ? If yes, see my code change suggestion above.

Reply:
I think the test was relying on fairly precise timing on how fast the local database would respond, but that precise timing was not the same when running in a GitHub action.  This change was just to make the test succeed in environments where the database command took a little longer to execute.